R/init.R

```diff
@@ -164,7 +169,139 @@ is_non_pkgdown_site <- function(dst_path) {
   }
 
   top_level <- dir_ls(dst_path)
-  top_level <- top_level[!path_file(top_level) %in% c("CNAME", "dev")]
+  top_level <- top_level[!path_file(top_level) %in% c("CNAME", "dev", "deps")]
 
   length(top_level) >= 1 && !"pkgdown.yml" %in% path_file(top_level)
 }
+
+create_bs_assets <- function(pkg) {
+  rlang::check_installed("htmltools")
+
+  # theme variables from configuration
+  bs_version <- pkg$bs_version
+  bootswatch_theme <- get_bootswatch_theme(pkg)
+
+  check_bootswatch_theme(bootswatch_theme, bs_version, pkg)
+
+  # variables from pkgdown defaults & user configuration
+  # user configuration takes precedence
+  sass_vars <- modify_list(
+    pkgdown_bslib_defaults(bs_version, bootswatch_theme),
+    pkg$meta$template$bslib
+  )
+
+  # first, defaults from bslib + pkgdown
+  bs_theme <- do.call(
+    bslib::bs_theme,
+    c(
+      list(version = bs_version, bootswatch = bootswatch_theme),
+      sass_vars
+    )
+  )
+
+  # Add body-color & navbar colors only in the absence of Bootswatch usage
```

Comment:
we want a different default for these variables and it's better to do it here I think rather than via adding our own CSS rules on top of Bootstrap.